index.html

```diff
@@ -9,9 +9,29 @@
 
     <link rel="stylesheet" href="styles.css" />
 
+    <script src="utlities.js" defer></script>
     <script src="app.js" defer></script>
+    
   </head>
   <body>
-      <h1>Surprise</h1>
+    <h1>Surprise</h1>
+
+    <form id="choose-ingredient-form">
+      <h2>Choose your favorite Ingredient</h2>
+      <input type="radio" id="strawberry" name="ingredient" value="strawberry"
```

Comment:
Let it be id="strawberryRadio"

Reply:
I remove the ids, the code is not really using them :)
one comment would have sufficed :)